week14/서빈/BOJ1516_게임개발.java

```diff
@@ -0,0 +1,76 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.StringTokenizer;
+
+/*
+ * java
+ * 276 ms
+ * 
+ * 위상 정렬
+ * 
+ * 위상 정렬을 통해 건물 완성 시간 계산
+ */
+
+public class BOJ1516_게임개발 {
+	static int N;
+	static int degree[], times[], result[];
+	static ArrayList<ArrayList<Integer>> list = new ArrayList<ArrayList<Integer>>();
+	static Queue<Integer> queue = new LinkedList<>(); // 위상 정렬을 위한 큐
+
+	public static void main(String[] args) throws Exception {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		N = Integer.parseInt(br.readLine());
+		degree = new int[N + 1];
+		times = new int[N + 1];
+		result = new int[N + 1];
+		StringTokenizer st = null;
+
+		for (int i = 0; i <= N; i++) {
+			list.add(new ArrayList<Integer>()); // 인접 리스트 초기화
+		}
+
+		for (int i = 1; i <= N; i++) {
+			st = new StringTokenizer(br.readLine());
+			times[i] = Integer.parseInt(st.nextToken()); // 각 건물 건설 시간 입력
+			int cnt = 0;
+			while (st.hasMoreTokens()) {
```

Comment:
아래에서 따로 break를 해주기 때문에 while문 조건을 true로 해도 좋을 것 같아요!
```suggestion
			while (true) {
```